tests/io_perfomance/eden.fio.tests.txt

```diff
@@ -0,0 +1 @@
+eden.escript.test -test.run TestEdenScripts/test_fio -args="gitrepo=eve-performance login=itmo-eve token=tocken"
```

Comment:
what is tocken?
I will run` ./eden  test ./tests/io_performance` ...
Where should I put token ? Some env?

Reply:
Having dealt with the tests, I temporarily redid this moment. And now the test will run deploy_app. This will produce results.

I also understand that it is important for Eden to know that the test has ended. Therefore, I will rewrite the test procedure on the next day by analogy with other tests, adding, for example, netcat to containers, so that Eden can understand whether the test has ended or not. Or maybe I will solve the problem with a timer. I'll fix that very soon.